lib/phoenix_inline_svg/helpers.ex

```diff
@@ -264,17 +219,34 @@ defmodule PhoenixInlineSvg.Helpers do
     |> Enum.into([])
   end
 
-  defp create_cached_svg_image({collection, name}, svgs_path) do
-    filename = name |> String.split(".") |> List.first
+  defp create_cached_svg_image({collection, name}) do
+    filename =
+      name
+      |> String.split("/")
+      |> List.last
+      |> String.split(".") 
+      |> List.first
+    svg = read_svg_from_path(name)
 
     quote do
+      def svg_image(unquote(filename)) do
```

Comment:
Should this function be defined in the macro? I would think this would just be a standard function in the module that uses the default collection to then call the [macro def that follows this one](https://github.com/nikkomiu/phoenix_inline_svg/pull/14/files?short_path=04c6e90#diff-e9df72ca9ba92ba50626afc237069177R236) like the [def on 102](https://github.com/nikkomiu/phoenix_inline_svg/blob/9e496c4ce1620cd500bb1bfaaf5b75ec723e3334/lib/phoenix_inline_svg/helpers.ex#L103):

```elixir
config_or_default(:default_collection, "generic")
```

Reply:
If this isn't defined in the macro, it doesn't cache the svg as a function. 

Also, the standard functions need to take a `@conn` or an `endpoint`